source/process.adoc

```diff
@@ -214,6 +214,8 @@ The Specification Committee works with the PMC to manage the overall vision for
 [[efsp-committee-approvals]]
 ==== Approvals
 
+Unless otherwise stated in this process (or a Working Group-specific derivative of this process), Super-majority approval is required for all decisions related to the development of specifications under this process.
```

Comment:
That obviously means "all decisions that are the purview of the Specification Committee", right?

Reply:
Yes. I'll change it to 

> Unless otherwise stated in this process (or a Working Group-specific derivative of this process), Super-majority approval is required for all decisions related to the development of Specifications under the purview of the Specification Committee.

Reply:
Per our discussion during the Specification Committee call today, I think that this may end up being far too broad. At this point, the process is no worse without it, so I'm going to remove it.